src/main/java/org/spongepowered/mod/mixin/entity/projectile/MixinEntityFishHook.java

```diff
@@ -206,4 +209,18 @@ public void setFishingRodItemStack(ItemStack fishingRod) {
         this.fishingRod = fishingRod;
     }
 
+    @Override
+    public void readFromNbt(NBTTagCompound compound) {
+        // TODO super.readFromNbt(compound);
+        if (compound.hasKey("damageAmount")) {
```

Comment:
Oops, must have accidentally removed the write when playing about with `ProjectileSourceSerializer`. I'll fix this